tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigProviderTest.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.eclipse.microprofile.config.tck;
 
+import java.io.*;
```

Comment:
I hate long lists of imports for obvious standard Java classes, but I'll fix it directly with a commit and update my IDE config appropriately just to stay consistent.